logsearch-jobs.yml

```diff
@@ -80,6 +84,43 @@ instance_groups:
       curator:
         purge_logs:
           retention_period: 180
+  - name: elastalert
+    release: elastalert
+    properties:
+      elastalert:
+        elasticsearch:
```

Comment:
Seems weird that the curator job on this node is talking to masters, but elastalert is talking to a copy of elastic running locally.  Can we use a single way of talking to elastic? (I don't really care which)